wolfcrypt/src/pkcs12.c

```diff
@@ -316,23 +316,23 @@ static int GetSafeContent(WC_PKCS12* pkcs12, const byte* input,
      * through the ContentInfo's and add them to our
      * AuthenticatedSafe struct */
     {
-        int CISz;
-        ret = GetSequence(input, &localIdx, &CISz, size);
+        word32 CISz;
+        ret = GetSequence(input, &localIdx, (int *)&CISz, size);
         if (ret < 0) {
             freeSafe(safe, pkcs12->heap);
             return ASN_PARSE_E;
         }
         CISz += localIdx;
-        while ((int)localIdx < CISz) {
-            int curSz = 0;
+        while (localIdx < CISz) {
+            word32 curSz = 0;
             word32 curIdx;
             ContentInfo* ci = NULL;
 
         #ifdef WOLFSSL_DEBUG_PKCS12
             printf("\t\tlooking for Content Info.... ");
         #endif
 
-            if ((ret = GetSequence(input, &localIdx, &curSz, size)) < 0) {
+            if ((ret = GetSequence(input, &localIdx, (int *)&curSz, size)) < 0) {
```

Comment:
Can't do this.
int is not always 32 bits.

Reply:
:heavy_check_mark: